nfs-ganesha/config/definitions/nfs-ganesha.yml

```diff
@@ -38,6 +38,11 @@
           default: "next"
 
       - string:
+          name: BRANCH
```

Comment:
Does this need to be exposed as a parameter? Why not just set ``BRANCH=$NFS_GANESHA_BRANCH`` in the ``build/setup``?

You'll also need to make sure you use the ``branch_slash_filter`` method from ``scripts/build_utils.sh`` on the branch value because a branch name with ``/`` in the name will break the shaman urls.